pyproject.toml

```diff
@@ -94,3 +94,11 @@ exclude_lines = [
 
 [tool.coverage.run]
 omit = ["*examples*"]
+
+filterwarnings =[
+  # JAX issues an over-eager warning if os.fork() is called when the JAX module is loaded, even if JAX isn't being used
+  'ignore:os\.fork\(\) was called\.:RuntimeWarning',
+
+  # Warning coming from blackjax
+  'ignore:jax\.tree_map is deprecated:DeprecationWarning',
+]
```

Comment:
This should be under [tool.pytest.ini_options] or in a pytest ini file if we use that here instead

Reply:
Done, thanks